### PR TITLE
Make items in LoadoutView draggable

### PR DESCRIPTION
**Pull request by bhollis**
This lets you drag items from existing loadouts into new loadouts (or loadouts you're editing) on the loadouts page. It's surprisingly useful.

### CODE REVIEW

**Review by nev-r — APPROVED** [review]
wait how long has loadout drawer accepted dragging... <_<

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ## Next
 
 * Collect Postmaster now requires an additional click to confirm.
+* You can drag items from existing loadouts in the Loadouts page into the loadout editor - use it to grab a couple of your favorite pieces from another loadout!
```
Comment (nev-r) [inline docs/CHANGELOG.md:4]:
```suggestion
* On the Loadouts page, you can now drag existing items on the page, into the current Loadout Editor. Use it to grab a couple of your favorite pieces from another loadout!
```
maybe? it was kind of a run-on string of prepositions. just me but maybe a good time to remind people this already works on inventory page as well.

Comment (bhollis) [reply docs/CHANGELOG.md:4]: Yeah that's a good idea

**Comment by bhollis** [comment]
> wait how long has loadout drawer accepted dragging... <_<

Since the very first version of DIM that had loadouts, I think

**Pull request merged**